About
----
AnnoTree is a web tool for visualization of genome annotations across large phylogenetic trees. AnnoTree version 1.2 includes [KEGG](https://www.genome.jp/kegg/), [PFAM](https://pfam.xfam.org/) and [Tigrfam](https://www.jcvi.org/tigrfams) annotations for over 30,000 bacterial and 1600 archaeal genomes. Phylogenetic and taxonomy information is derived from the [GTDB](http://gtdb.ecogenomic.org/) database (Release R95).

Recommended Browser
---
For the best result, we recommend using a **Chrome** browser.

Team
---
AnnoTree was developed by the Doxey Lab at the University of Waterloo.
 
* Han Chen (core developer)
* Kerrin Mendler (core developer)
* Huagang Tan (core developer)
* Briallen Lobb (tool design)
* Donovan Parks (GTDB genomic data and TOL)
* Laura Hug (phylogenomics, case studies, TOL)
* Andrew Doxey (tool conception/design)

Citation
----

Mendler K, Chen H, Parks DH, Hug LA, Doxey AC. (2019) AnnoTree: visualization and exploration of a functionally annotated microbial tree of life. Nucleic Acids Research 47:4442-4448; doi: http://dx.doi.org/10.1093/nar/gkz246


Contact us
---
Please email Andrew Doxey (acdoxey@uwaterloo.ca).

Version
----
The version for this app and its associated data are listed under the main app footer.
